### ISSUE TITLE
How to make chat head appear before prefix on chat?

### ISSUE THREAD

**Issue by Kloytz** [dzwdz/chat_heads#110]
![image](https://github.com/user-attachments/assets/c9f1eb7e-8138-47c4-98a5-cdf5290667b7)
For some reason prefix appears before the chat head on the chat but on tab it appears after the head

**Comment by Fourmisain** [comment]
You can change the render position to "Before Line", so it'll render at the beginning of the line just like in the tab list, though It'll before everything, including the brackets.

Does the tab list show `MEMBERKloytz` (without space) _exactly_ or is it different?
Chat Heads should match the tab name and put the head in front of it, but only if they're exactly the same.
It looks like it matched `Kloytz` (your profile name?) instead.

You don't happen to know what server plugins were used?

**Comment by Kloytz** [comment]
Yep after creating the github i checked the options and now im using before the lines.

Heres how it appears on tab:
![image](https://github.com/user-attachments/assets/e4282d10-7f41-443d-8bef-92ce1090e933)

**Comment by Kloytz** [comment]
Also, i have no plugins, playing singleplayer to make a server on fabric.

**Comment by Fourmisain** [comment]
Ah, with that tab name it's no wonder it doesn't detect it, all that ` [12💀] ❤️❤️❤️...` stuff is part of the name too and since that doesn't appear in the chat message, Chat Heads doesn't match it. (Also it says `OWNER` instead of `MEMBER` but I assume that changed on both ends.)

That `MEMBER` / `OWNER` stuff is most likely done with custom font like the chat emojis are, in other words, it's just text. If you really want to you could figure out what that text is and add a name alias in the Chat Heads config (`<THAT TEXT>Kloytz -> Kloytz`), so it would match it as your name and put the head at the right place.

I don't think there's a way to "fix" this, aka automatically do this, as there's no way of telling what part of the tab name is supposed to the _actual_ name and what should be ignored.
Well, except I hardcode the `MEMBER` / `OWNER` stuff in, but for that I'd need to know what ~~plugin~~ mod it actually is that's doing that.

**Comment by Kloytz** [comment]
The mod is style player list, yes the [12💀] is part of the name via a code and from the hearts part is just /scoreboard command.

member/owner stuff is as you said a font via resourcepack.
https://modrinth.com/resourcepack/nautical-ranks
